version.c

```diff
@@ -5,6 +5,20 @@
 const char git_version_string[] = GIT_VERSION;
 const char git_built_from_commit_string[] = GIT_BUILT_FROM_COMMIT;
 
+/*
+ * TODO: ensure consecutive non-printable characters are only replaced once;
+ * Trim and replace each character with ascii code below 32 or above
+ * 127 (included) using a dot '.' character.
+*/
+static void redact_non_printables(struct strbuf *buf)
```

Comment:
As the name of the function is now redact_non_printables(), the subject of the commit message should use that name instead of strbuf_sanitize(). So it should be something like "version: refactor redact_non_printables()".

Reply:
Yeah, this is true. I will change it.